src/en/space-station-14/player-interaction/proposals/night-vision.md

```diff
@@ -0,0 +1,25 @@
+# Night Vision
+
+| Designers | Implemented |
+|---|---|---|
+| pofitlo | :white_check_mark: No |
+
+
+## Overview
+
+adds night vision, which is a dimly lit area around the character that only he sees.
```

Comment:
This could probably be renamed to darksight to match up with what the same feature is called in SS13